### PR TITLE
Fix storybook

### PR DESCRIPTION
**Pull request by juancwu**
Now storybook will merge the config from `vite.config.js` and use the same configurations.

### CODE REVIEW

**Review by aidantrabs — APPROVED** [review]
^--^

File: .storybook/main.cjs [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
+        '@storybook/addon-interactions',
+    ],
+    framework: '@storybook/react',
+    core: {
+        builder: '@storybook/builder-vite',
+    },
+    features: {
+        storyStoreV7: true,
+    },
+    async viteFina(config) {
```
Comment (SherRao) [inline .storybook/main.cjs:20]: Is this function name a typo?

**Review by SherRao — COMMENTED** [review]
TODO: fix typo?

File: .storybook/main.cjs (hunk 1 above)
Comment (juancwu) [reply .storybook/main.cjs:20]: bruh wtf, thanks for letting me know

Comment (SherRao) [reply .storybook/main.cjs:20]: Also might be worth adding some documentation to this file to explain why we're merging the vite config file.

Comment (aidantrabs) [reply .storybook/main.cjs:20]:
```
your mom
```

**Pull request merged**